data/credits/contributors.json

```diff
@@ -0,0 +1 @@
.githubusercontent.com/u/60521810?v=4","profile":"https://scratch.mit.edu/users/SuperScratcher_1234","contributions":["doc"],"commits":1},{"login":"ChrisIsBadAtFnF","name":"ChrisIsBadAtFnF","avatar_url":"https://avatars.githubusercontent.com/u/90082208?v=4","profile":"https://github.com/ChrisIsBadAtFnF","contributions":["example"]},{"login":"BarelySmooth","name":"BarelySmooth","avatar_url":"https://avatars.githubusercontent.com/u/83728060?v=4","profile":"https://github.com/BarelySmooth","contributions":["example","code"]},{"login":"Chiroyce1","name":"Chiroyce","avatar_url":"https://avatars.githubusercontent.com/u/97374054?v=4","profile":"https://scratch.mit.edu/users/Chiroyce","contributions":["doc","code"]},{"login":"Evrick14","name":"Evrick_14","avatar_url":"https://avatars.githubusercontent.com/u/99140357?v=4","profile":"https://scratch.mit.edu/users/Evrick_14/","contributions":["translation"]},{"login":"Norbiros","name":"Norbiros","avatar_url":"https://avatars.githubusercontent.com/u/89837724?v=4","profile":"http://bestopia.pl","contributions":["translation"]},{"login":"diaowinner","name":"diaowinner","avatar_url":"https://avatars.githubusercontent.com/u/32033700?v=4","profile":"https://github.com/diaowinner","contributions":["translation"]},{"login":"Dhuls-Scratch","name":"dhuls","avatar_url":"https://avatars.githubusercontent.com/u/87714852?v=4","profile":"https://github.com/Dhuls-Scratch","contributions":["code"]},{"login":"RIMOPA","name":"RIMOPA","avatar_url":"https://avatars.githubusercontent.com/u/97112832?v=4","profile":"https://scratch.mit.edu/users/RIMOPA/","contributions":["example","ideas"]},{"login":"Jazza-231","name":"Jazza-231","avatar_url":"https://avatars.githubusercontent.com/u/81956724?v=4","profile":"https://github.com/Jazza-231","contributions":["ideas"]},{"login":"R081N-08","name":"VeryBlueMan","avatar_url":"https://avatars.githubusercontent.com/u/88439743?v=4","profile":"https://github.com/R081N-08","contributions":["translation"]},{"login":"Carsvroom","name":"Carsvroom","avatar_url":"https://avatars.githubusercontent.com/u/84688137?v=4","profile":"https://github.com/Carsvroom","contributions":["mod"]},{"login":"shock59","name":"Shock59","avatar_url":"https://avatars.githubusercontent.com/u/42130427?v=4","profile":"https://scratch.mit.edu/users/shock59","contributions":["mod","video"]},{"login":"8bitjake","name":"8BitJake","avatar_url":"https://avatars.githubusercontent.com/u/75950907?v=4","profile":"https://scratch.mit.edu/users/8bitjake","contributions":["mod"]},{"login":"Aid0nModder","name":"Aidan","avatar_url":"https://avatars.githubusercontent.com/u/71894636?v=4","profile":"https://github.com/Aid0nModder","contributions":["mod","video"]},{"login":"ChangedRuby","name":"ChangedRuby","avatar_url":"https://avatars.githubusercontent.com/u/65453800?v=4","profile":"https://github.com/ChangedRuby","contributions":["mod"]},{"login":"ImgBotApp","avatar_url":"https://avatars.githubusercontent.com/u/31427850?v=4","commits":17}]
```

Comment:
Why are you not fetching this from the repo?

Reply:
I think it is accidentally pushed. Will remove this.